pajbot/web/common/assets.py

```diff
@@ -30,14 +30,24 @@ def init(app):
     assets.register("pleblist_host", pleblist_host)
 
     # CLR Overlay
-    # Availabe under: clr_overlay_js, clr_overlay_css, clr_shared_js
-    clr_overlay_js = Bundle("scripts/clr.overlay.js", filters="jsmin", output="gen/scripts/clr.overlay.%(version)s.js")
+    # Availabe under: clr_overlay_js, clr_overlay_css, clr_donations_js, clr_donations_css, clr_shared_js
```

Comment:
please undo changes made in this file, must've been some merge magic that they made it in